MapboxCoreNavigation.podspec

```
@@ -40,7 +40,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 81.0"
+  s.dependency "MapboxNavigationNative", "~> 82.0.1"
```

Please set only minor version like:
```suggestion
  s.dependency "MapboxNavigationNative", "~> 82.0"
```